### DISCUSSION TITLE
About ABDL question line

### DISCUSSION THREAD

**Discussion by Coffin123** [zippity21/Stardew_Valley_Regression_Mod#54] [Q&A]
Hello!

Love your mod since I found it!
Tho I don't know why but I couldn't get any new dialogs with Vins or Jan.

I'm playing the latest mod version and using SMAPI beta
Also I did change the json so pooping is false.
Introduction quest is not finished.

Is that what is causing it?

Thank you for your time in advance!

**Comment by zippity21** [comment]
Thank you for your kind words.

This may be better suited for the issues page since it's about functionality that appears to be broken. So I'm going to move it, and we'll continue from there.

Is the issue only with Vins and Jan, or all ABDL dialogue?

Also, what version of the mod are you using? The most recent version should be enforcing the new non-beta SMAPI release. When you start the game, the SMAPI terminal should notify you of an available update for installed mods.

**Reply by Coffin123** [reply]
Hi.

Thank you for your reply!
I just found out that I had SMAPI-beta and didnt have the dialog mod, that was the problem.
Used SMAPI-Official and added the dialog-mod and now it works like a charm!

Thank you for your help, and keep up the good work!!

**Reply by zippity21** [reply]
I'm glad you got it working. 😃

While I've been doing updates and bug fixes, I can't take all the credit. The dialogue has been done mostly by @TikkuTokku . So I'm grateful to all contributors and of course the original author.

**Reply by TikkuTokku** [reply]
Awww, thank you kindly!
